### ISSUE TITLE
Still empty file after 5 hrs of waiting.

### ISSUE THREAD

**Issue by manolito62** [MarvNC/mangadex-list-exporter#4]
![Untitled](https://user-images.githubusercontent.com/4824412/104822753-cba51900-57f9-11eb-9787-eb7a6e7e9785.png)

Still the same problem, and this time it **took 5 hrs** to show the empty results, as you see in the pic, there is the console, the empty file, and is firefox 84.0.2 (64-bit) and using Greasemonkey 4.10 Release.

#3

**Comment by MarvNC** [comment]
I verified the issue with greasemonkey on FF, for now I might suggest using Violentmonkey as it works for this script and [Greasemonkey won't find as much support for some scripts](https://www.greasespot.net/2017/09/greasemonkey-4-for-users.html). I'll work on fixing the issue in the meantime.

**Comment by MarvNC** [comment]
Fixed in https://github.com/MarvNC/mangadex-list-exporter/commit/4985e075ccbee0e3157d10913bfdb0a6554e6ae0, it was a dumb mistake on my part. Though I would still recommend switching to Violentmonkey. Let me know if it works, though you may want to test it on an alt account first if your list is so long.

**Comment by manolito62** [comment]
Fun thing, my list is not even complete, is missing a few mangas here and ther (over 150), gonna test it out right now, see you in 5 hrs or more.

Changed delay to const DELAY = 500; to do it a bit faster.

**Comment by manolito62** [comment]
Kay, is me again, it finally worked fine, the file is not empty anymore, i got 3 errors in the console, also not sure if because i set it to fast in the script or the website.
![1](https://user-images.githubusercontent.com/4824412/104825183-3ca1fc00-580d-11eb-8d32-68fbec2f4ab1.PNG)
![2](https://user-images.githubusercontent.com/4824412/104825184-3dd32900-580d-11eb-8291-1f2e78fb461c.PNG)
![3](https://user-images.githubusercontent.com/4824412/104825185-3dd32900-580d-11eb-8546-646a25df1c34.PNG)

Other than that, i just have to test it in MAL to see if works.

Also, how i can do this NOT BY HAND?
![4](https://user-images.githubusercontent.com/4824412/104826938-d49fe580-580d-11eb-96a5-733c1e4fa1b2.PNG)

**Comment by MarvNC** [comment]
Yeah, mangadex isn't too reliable and I've experienced ratelimiting and temporary bans by doing delays under 750ms, which is why it's set to 1000ms by default. It's even less reliable now honestly, and as you can see in those errors it was ddos guard, you might have to go update those manga manually for now. To change all your update_on_import values to true, open the xml in your favorite text editor (such as notepad++ or vscode), ctrl + H, and put in `<update_on_import>0</update_on_import>` and replace with `<update_on_import>1</update_on_import>`.

**Comment by manolito62** [comment]
Perfect, worked perfectly, also, you should add an output of file or a message showing what manga could not be imported, like those 3, also, it worked properly, it only say it update 1400 manga in MAL (more or less) , but now i have 1956 entries in there (before it was almost half of that), so it worked properly, thanks. (also worked for anilst), thank you so much for this script, i think you should change the update import to 1, just saying, maybe you have a reason of why not, and at last, a bit of flexing:
![123](https://user-images.githubusercontent.com/4824412/104828538-5ea47a00-581f-11eb-9603-90a8dde8e06a.PNG)
Thought since around 2 or 3 years ago i reduce the speed on how i see anime, now i barely see from 1 to 5 per season. 

TL;DR: you can count this case as close =3

**Comment by MarvNC** [comment]
I left file import at 0 because MD chapter counts can be very inaccurate in my experience and also because MD counts chapters differently than other sites, I figured it'd be best to leave it to the user to enable it instead. I might put a guide on changing it to 1 now, though. And the the manga that error'd, I'll work on something for that in the future. Thanks for the feedback.
